packages/flutter_tools/templates/app/android.tmpl/gradle.properties

```diff
@@ -1 +0,0 @@
-org.gradle.jvmargs=-Xmx1536M
```

Comment:
Why is this getting deleted?  I believe this was put in place to fix a frequent issue where gradle would crash due to lack of memory.

Reply:
It's not being deleted - it moved from a static file to a template.

Reply:
I see it now, sorry about that.